master/pyproject.toml

```diff
@@ -1,7 +1,11 @@
+# sqlfluff defaults: https://docs.sqlfluff.com/en/2.1.4/configuration.html#default-configuration
+
 [tool.sqlfluff.core]
 templater = "jinja"
 sql_file_exts = ".sql,.sql.j2,.dml,.ddl"
-exclude_rules = "L034,layout.spacing"
+exclude_rules = ""
+max_line_length = 100
+output_line_length = 100
```

Comment:
my take
```suggestion
max_line_length = 140
output_line_length = 140
```

Reply:
went off of what we have for go but sgtm